roles/sarek/templates/apply_recalibration.sh.j2

```diff
@@ -0,0 +1,59 @@
+#! /bin/bash -l
+
+################################################################################
+##
+## This is a bash script for applying pre-computed recalibration statistics
+## on a BAM file and generate a Base Quality Score Recalibrated (BQSR) BAM file.
+## This is done with the PrintReads program of GATK.
+##
+## To use the script, pass the BAM file as the first argument to the script and
+## the recalibration statistics file (output from GATK's BaseRecalibrator) as
+## the second argument. The output BAM file name is created by stripping the .bam
+## file extension and replacing it with .recal.bam.
+##
+## Example:
+##   bash applyRecalibration.sh /path/to/bamfile.bam /path/to/recalibration.table
+##
+## To submit the script as a job to a SLURM cluster, just submit with the
+## UPPNEX id of your project, e.g.:
+##   sbatch -A b2015999 apply_recalibration.sh /path/to/bamfile.bam \
+##     /path/to/recalibration.table
+##
+################################################################################
+
+#SBATCH -p core
+#SBATCH -n 2
+#SBATCH -N 1
+#SBATCH -J apply_recalibration
+#SBATCH -t 48:00:00
+#SBATCH -o apply_recalibration.%j.log
+#SBATCH -e apply_recalibration.%j.log
+
+# exit on error
+set -e
+set -o pipefail
+
+# inputs
+BAMIN="${1}"
+RECALTABLE="${2}"
+
+# if running on a compute node, use the scratch area for output, otherwise write
+# directly to the output directory
+if [ -z "$SLURM_JOB_ID" ]
+then
+  SNIC_TMP="`dirname ${BAMIN}`"
+fi
+
+BAMOUT="$SNIC_TMP/`basename ${BAMIN/.bam/.recal.bam}`"
+FASTAREF="/sw/data/uppnex/ToolBox/hg38bundle/Homo_sapiens_assembly38.fasta"
```

Comment:
I have no good suggestion on how to do it, but it would be nice if the FASTAREF could be dynamic since not all projects will be mapped against 38. Perhaps having the reference as an optional third script argument with this value as the default?

Reply:
Ah, very good point! I'll add it as an argument

Reply:
@b97pla Updated now, would be great if you could have another look.

Reply:
:+1: looks good